woocommerce/woocommerce-admin
# Add Report CSV Export Endpoint

Addresses #311 partially.

This PR adds:

- 2 new endpoints: `/reports/[type]/export` and `/reports/[type]/export[id]/status`
- Export file download capability (on `admin_init`, like core's product export)
- Queued batch creation of report export files
- Example filter for modifying schema when adding fields to report API responses

### Detailed test instructions:

- `POST /wc/v4/reports/products/export`
- `GET /wc/v4/reports/products/export/<id here>/status` (get URL from `POST` response)
- When 100%, `download_url` should be in status response
- Open download URL in browser
- Verify CSV contents
